lib/ex_machina/ecto_strategy.ex

```diff
@@ -3,10 +3,20 @@ defmodule ExMachina.EctoStrategy do
 
   use ExMachina.Strategy, function_name: :insert
 
-  def handle_insert(%{__meta__: %{__struct__: Ecto.Schema.Metadata}} = record, %{repo: repo}) do
-    repo.insert! record
+  def handle_insert(%{__meta__: %{state: :loaded}} = record, _) do
+    raise ArgumentError,
```

Comment:
I think `raise` without `ArgumentError` is ok. A `RuntimeError` should be fine here